lib/firebase/src/user.dart

```diff
@@ -85,6 +100,9 @@ sealed class ThcUser {
   final bool registered;
   final bool? notify;
   final String? profilePictureUrl;
+  bool? isAudioEnabled;
+  bool? isVideoEnabled;
```

Comment:
Unfortunately, adding these non-`final` values to an `@immutable` class is triggering some static analysis problems.

In my opinion, it would make the most sense to store these values in `LocalStorage`. What are your thoughts?